src/DIRAC/FrameworkSystem/Service/TokenManagerHandler.py

```diff
@@ -59,7 +59,7 @@ def initializeHandler(cls, *args):
         try:
             cls.__tokenDB = TokenDB()
```

Comment:
Shouldn't TokenDB get the logger?

Reply:
Yeah, `TokenDB` is part of a few special DB that do not have a default constructor, so I should modify it properly... I'll do that in this PR